src/track/serato/markers2.h

```diff
@@ -364,12 +331,11 @@ QDebug operator<<(QDebug dbg, const SeratoMarkers2LoopEntry& arg) {
 // https://github.com/Holzhaus/serato-tags/blob/master/docs/serato_markers2.md
 //
 class SeratoMarkers2 final {
-public:
+  public:
     SeratoMarkers2() = default;
     explicit SeratoMarkers2(
             QList<std::shared_ptr<SeratoMarkers2Entry>> entries)
-        : m_allocatedSize(0)
-        , m_entries(std::move(entries)) {
+            : m_allocatedSize(0), m_entries(std::move(entries)) {
```

Comment:
We use one initialization per line.
I guess this was changed by an autoformater because of the precising ,